templates/admin-page.php

```diff
@@ -31,7 +31,7 @@
 					<?php } ?>
 				</select>
 
-				<input type="submit" value="<?php esc_attr_e( 'Check it!', 'plugin-check' ); ?>" />
+				<input type="submit" value="<?php esc_attr_e( 'Check it!', 'plugin-check' ); ?>" id="pc_check_it" />
```

Comment:
See above. More importantly though, let's not use `check_it`, that's not really a good description of the element. More something like "submit" or "submit button". How about `plugin-check__submit`?